schemas/src/elements/inputs/Input.ChoiceSet.json

```diff
@@ -20,10 +20,14 @@
       "type": "string",
       "description": "The initial choice (or set of choices) that should be selected. For multi-select, specify a comma-separated string of values."
     },
+    "placeholder": {
+      "type": "string",
+      "description": "Description of the input desired. Displayed when no selection has been made, style is compact and multi selection is false."
```

Comment:
Thanks for adding this Joseph! Can you tweak the format a bit, something like: "Description of the input desired. Only visible when no selection has been made, the `style` is `compact` and `isMultiSelect` is `false`."

Reply:
sure

---
In reply to: [494550835](https://github.com/microsoft/AdaptiveCards/pull/4811#discussion_r494550835) [](ancestors = 494550835)